programming-core/src/main/java/org/objectweb/proactive/core/runtime/ProActiveRuntimeImpl.java

```diff
@@ -834,6 +834,13 @@ public void unregisterAllVirtualNodes() {
     @Override
     public List<UniversalBody> getActiveObjects(String nodeName, String className) {
         // the array to return
+
+        Class requiredClass = null;
+        try {
+            requiredClass = Class.forName(className);
```

Comment:
Note that now the getActiveObjects(String) method needs to take in parameter the full name of the class (including packages).